Library/Tracking/KSRAnalytics.swift

```diff
@@ -1458,6 +1506,19 @@ private func contextProperties(
   return result.prefixedKeys(prefix)
 }
 
+// MARK: - Interaction Properties
+
+private func interactionProperties(
+  targetUserId: String,
+  prefix: String = "interaction_"
+) -> [String: Any] {
+  var result: [String: Any] = [:]
+
+  result["target_uid"] = targetUserId
+
+  return result.prefixedKeys(prefix)
```

Comment:
Why not just set `result["interaction_target_uid"] = targetUserId`?

Reply:
We could do that, but I opted to use the same pattern we're using elsewhere just in case we want to add more Interaction specific properties moving forward.